### ISSUE TITLE
this._documents.create is not a function

### ISSUE THREAD

**Issue by rstokes** [interfax/interfax-nodejs#5]
Error sending large files.  It seems that when the file size exceeds the chunk size we are receiving the below error.

TypeError: this._documents.create is not a function
    at File.initializeDocument (/Users/rob/repos/narus-web/backend/node_modules/interfax/lib/file.js:79:23)
    at File.initializeBinary (/Users/rob/repos/narus-web/backend/node_modules/interfax/lib/file.js:51:21)
    at new File (/Users/rob/repos/narus-web/backend/node_modules/interfax/lib/file.js:33:12)
    at Files.create (/Users/rob/repos/narus-web/backend/node_modules/interfax/lib/files.js:38:18)
    at Object.module.exports.sendSecureFax.err [as sendSecureFax] (/Users/rob/repos/narus-web/backend/services/reporting/fax.js:39:27)
    at module.exports.shareReport.box.request.then.then.report (/Users/rob/repos/narus-web/backend/services/reporting/index.js:176:20)

It appears to be triggered when this condition is true in initializeBinary.

```javascript
function initializeBinary(data, mimeType) {
      if (data.length > this._chunkSize) {
        return this.initializeDocument(data, mimeType);
      }
      this.header = 'Content-Type: ' + mimeType;
      this.body = data;
      this._triggerReady(true);
}
```

**Comment by cbetta** [comment]
Hey @rstokes thanks for reporting this. I found the bug, fixed it, and updated the docs. A little heads up that `files.create` uses a promise. It incorrectly suggested in the docs to return a file instance.

**Comment by cbetta** [comment]
PS: I released 0.8.1 to NPM which has the fix.

**Comment by rstokes** [comment]
Thank you.  This fixed our issue.
